src/containers/header/Header.js

```diff
@@ -94,30 +52,28 @@ export const UnconnectedHeader = ({
     if (provider === 'facebook') {
```

Comment:
This can be written better:

authData.token_provider = provider === 'facebook' && 'facebook'